tampermonkey/PPT_AutoComplete.js

```diff
@@ -1,7 +1,7 @@
 // ==UserScript==
 // @name         PPT Registration AutoComplete
 // @namespace    http://tampermonkey.net/
-// @version      15.2
+// @version      15.3
```

Comment:
is this the only change to the script?

Reply:
My mind must have wandered off. I thought I needed to make some changes but I decided that I did not. I'll revert.